src/shared/components/mutationTable/column/ProteinChangeColumnFormatter.tsx

```diff
@@ -95,4 +184,69 @@ export default class ProteinChangeColumnFormatter {
 
         return content;
     }
+
+    private static getProteinChangeData(
```

Comment:
This function has a lot of custom logic in it. Might be better to move into a utility file and add a few tests.

Reply:
@onursumer good point. There were some tests for `MutationAnnotator`, we might also update them to fit the new changes

Reply:
we removed the logic here, so no need for tests